torch/_inductor/cpu_vec_isa.py

```diff
@@ -174,7 +174,7 @@ class VecSVE(VecISA):
         "CPU_CAPABILITY_SVE256",
         "AT_BUILD_ARM_VEC256_WITH_SLEEF",
     ]
-    _arch_flags = "-march=armv8-a+sve -msve-vector-bits=256"
+    _arch_flags = "-march=armv8.2-a+sve -msve-vector-bits=256"
```

Comment:
do we need `-march=armv8.2-a+sve+bf16` if the platform supports `bf16` intrinsics for `torch.compile` to work ? 
request: an API check `torch.cpu._is_arm_sve_bf16_supported` to enable it? but this seems too convoluted already 👀

Reply:
Yes better to add a compilation check for bf16.

I see cpuinfo submodule already has this API implemented **cpuinfo_has_arm_sve_bf16(void)** .
https://github.com/pytorch/cpuinfo/blob/main/include/cpuinfo.h#L2039

You would have to implement/introduce it as a python API in PyTorch and then would be able to use it in [torch/_inductor/cpu_vec_isa.py](https://github.com/pytorch/pytorch/pull/143666/files/8e5134e9c22cdb6150e425bee43015998ae55c59#diff-39665ae5ca878523e2f73397eec7080a5eff86c46c6bd01377c0be235d97109c)

I have introduced an API using similar approach namely - _is_arm_sve_supported() in the below commit
https://github.com/pytorch/pytorch/pull/138388/commits/b043609db44069eb442ea4b754fe8f8a433e9c06#

You can use it as reference to implement the API for SVE_BF16 and add the compilation check.